chainntnfs/bitcoindnotify/bitcoind_test.go

```diff
@@ -221,7 +222,7 @@ func testHistoricalConfDetailsNoTxIndex(t *testing.T, rpcpolling bool) {
 	unknownConfReq, err := chainntnfs.NewConfRequest(&unknownHash, testScript)
 	require.NoError(t, err, "unable to create conf request")
 	broadcastHeight := syncNotifierWithMiner(t, notifier, miner)
-	_, txStatus, err := notifier.historicalConfDetails(
+	txConfirmation, txStatus, err := notifier.historicalConfDetails(
```

Comment:
getting txConfirmation here will increase the coverage by verifying the output results.